### DISCUSSION TITLE
Missing file

### DISCUSSION THREAD

**Discussion by giftcp** [meld-cp/obsidian-build#1] [Q&A] [answered]
I cannot find the main.js file in your repository.

The plugin is also not published in the community plugins

**Comment by meld-cp** [comment, marked as answer]
Hi @giftcp , it's awaiting the signoff for it to be added to the community plugins list.  In the meantime you can get the assets via the releases page: https://github.com/meld-cp/obsidian-build/releases

**Comment by giftcp** [comment]
got it!

[image: image.png]

On Wed, Feb 8, 2023 at 12:59 AM meld-cp ***@***.***> wrote:

> Hi @giftcp <https://github.com/giftcp> , it's awaiting the signoff for it
> to be added to the community plugins list. In the meantime you can get the
> assets via the releases page:
> https://github.com/meld-cp/obsidian-build/releases
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/meld-cp/obsidian-build/discussions/1#discussioncomment-4898880>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AECOTDNYWJGW6CMK6MBCSA3WWLHVLANCNFSM6AAAAAAUUP46TQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Gift
